### PR TITLE
Implement popup windows

### PR DESCRIPTION
**Pull request by TyGriffin115**
Implemented Popup Warnings for every Delete Button in the front-end. The delete functionality for the scenario delete buttons on the logistics page and dashboard page have yet to be implemented. We should wait until after the integration is complete for the logistics and dashboard pages. 

Gif1

Gif12

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ConfigurationSideBarComponents/NavSideBarNode.js [hunk 1]
```diff
@@ -77,15 +74,7 @@ export default function NavSideBarNode(props) {
...
-                    <Grid item xs={2} className={classes.deleteButtonContainer}>
-                        <Button
-                            className={classes.deleteButton}
-                            color="primary"
-                            onClick={handleDelete}
-                        >
-                            <DeleteForeverIcon />
-                        </Button>
-                    </Grid>
+                    <DeletePopupSidebar id={id} deleteByID={deleteByID} />
```
Comment (eneadodi) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ConfigurationSideBarComponents/NavSideBarNode.js]: Nice job working with the current implementation!

**Review by eneadodi — DISMISSED** [review]
There probably was a way to implement the popup so you didn't have to make a different component for each page type. Nevertheless it seems to work out, just was more work for you. I approve!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ConfigurationSideBarComponents/NavSideBarNode.js (hunk 1 above)
Comment (TyGriffin115) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ConfigurationSideBarComponents/NavSideBarNode.js]: Fixed! There are only two files now, one for a generic delete warning and the other is a warning for deleting an entire scenario and all its data.

**Review by eneadodi — DISMISSED** [review]
Looks good! Well done.

**Review by Enoch-Hsiao — APPROVED** [review]
LGTM

**Pull request merged**